server/src/main/java/com/testsigma/controller/api/agent/AgentDevicesController.java

```diff
@@ -54,6 +59,8 @@ public class AgentDevicesController {
   private final StorageServiceFactory storageServiceFactory;
   private final ProvisioningProfileDeviceService provisioningProfileDeviceService;
   private final TestsigmaOSConfigService testsigmaOSConfigService;
+  private static final String WDA_APP_FILEPATH = "https://s3.amazonaws.com/ios.testsigma.com/wda/wda_simulator.ipa";
```

Comment:
@tarun-testsigma can we have these URLs fetch on demand from os services?

Reply:
Will do that

Reply:
Done